README.md

```diff
@@ -17,6 +17,13 @@ cd common-metrics
 mvn install
 ```
 
+Install root-pom found in the iamp-commons root directory
+
+```
+cd ..
+mvn install
+```
+
```

Comment:
Running mvn install in the root directory appears to try to install all of the sub-directory pom.xml files in the correct order. However, it errors partway through and can't install common-security. It tries to pull spring-security-oauth2-autoconfigure:jar:2.1.0.M4 and fails. bcparis-service still builds fine locally without installing the common-security pom, but I wanted to make a note about this.